test/NuGet.Core.Tests/NuGet.ProjectModel.Test/JsonPackageSpecReaderTests.cs

```diff
@@ -27,13 +27,14 @@ public void PackageSpecReader_PackageMissingVersion()
         {
             // Arrange
             var json = @"{
-                          ""dependencies"": {
-                                ""packageA"": {
-                                    ""type"": ""build""
-                                }
-                            },
                             ""frameworks"": {
-                                ""net46"": {}
+                                ""net46"": {
+                                  ""dependencies"": {
```

Comment:
What happens if dependencies node is missing?

Reply:
empty dependencies, no error is raised.